specs/jsonschema-validation.md

```diff
@@ -88,16 +88,27 @@ specification.
 ## Meta-Schema {#meta-schema}
 
 The current IRI for the default JSON Schema dialect meta-schema is
-`https://json-schema.org/draft/next/schema`. For schema author convenience, this
-meta-schema describes a dialect consisting of all keywords defined in this
+`https://json-schema.org/1/2025`. This IRI encodes the specifications version
+and release year. Because all schemas written to conform to a given version are
```

Comment:
I feel like there's some ambiguity here. You've defined the "version" as the iteration + release year. But, here you're calling the iteration "specification version".

```suggestion
`https://json-schema.org/1/2025`. This IRI encodes the specifications iteration
and release year. Because all schemas written to conform to a given version are
```